integration-tests/src/test/java/oracle/weblogic/kubernetes/ItStickySession.java

```diff
@@ -191,14 +161,6 @@ public static void init(@Namespaces(4) List<String> namespaces) {
 
   @AfterAll
   void tearDown() {
```

Comment:
if traefik is installed in a different namespace, it will be uninstalled by default. You can remove teardown() method here.

Reply:
removed